dom.html

```diff
@@ -0,0 +1,232 @@
+            <textarea class="description" rows="4"></textarea>
+
+            <div class="flex-container">
+              <div>Click to Save</div>
+              <button type="submit">Save</button>
+            </div>
+          </form>
+        </div>
+        <div class="filter-input">
+          <label for="filter">Search by Company Name:</label>
+          <input type="text" id="filter" oninput="filterExperiences()" />
+        </div>
+
+        <div class="experiences-list">
+          <!-- Experience entries will be dynamically added here -->
+        </div>
+      </div>
+
+      <div class="sidebar">
+        <div>
+          <img src="New Project (40).webp" alt="Avatar" class="avatar" />
+        </div>
+        <div class="skill">
+          <h1>Fazila Abbasi</h1>
+          <h2>Skill's</h2>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>MS Word</li>
+            <li>MS Excel</li>
+            <li>Power Point</li>
+            <li>Java Script</li>
+            <li>Team Working</li>
+            <li>Documentation</li>
+            <li>Event Planning</li>
+            <li>Project Reports</li>
+            <li>Communication Skills</li>
+            <li>Presentation Skills</li>
+            <li>Project Management</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <footer>
+      <div class="footer">
+        <a href="https://www.linkedin.com/in/fazila-abbasi-a07b18264/"
+          >Link to My LinkedIn Profile!</a
+        >
+      </div>
+    </footer>
+    <script>
+      const storedExperiences =
+        JSON.parse(localStorage.getItem("experiences")) || [];
+
+      for (const experience of storedExperiences) {
+        displayExperience(experience);
+      }
+
+      document
+        .querySelector(".add-experience")
+        .addEventListener("click", function () {
+          document.querySelector(".experience-form").style.display = "flex";
+        });
+
+      document
+        .querySelector(".add-experience-form")
+        .addEventListener("submit", function (event) {
+          event.preventDefault();
+
+          const companyName = document.querySelector(".company-name").value;
+          const startDate = document.querySelector(".start-date").value;
+          const endDate = document.querySelector(".end-date").value;
+          const description = document.querySelector(".description").value;
+
+          const newExperience = {
+            companyName: companyName,
+            startDate: startDate,
+            endDate: endDate,
+            description: description,
+          };
+
+          storedExperiences.push(newExperience);
+          localStorage.setItem(
+            "experiences",
+            JSON.stringify(storedExperiences)
+          );
+
+          displayExperience(newExperience);
+
+          document.querySelector(".add-experience-form").reset();
+          document.querySelector(".experience-form").style.display = "none";
```

Comment:
All of your document selectors should be in a variable declared on top of its scoped area. Please check the rest of the files.

Reply:
fix

Reply:
![image](https://github.com/FazilaAbbasi/Fazila-HTML-CSS-Basics/assets/72794697/9ed5cd10-9426-4320-afbc-8c4604d7b418)

This is still not done. 
This should be in a variable as its being used in multiple places.